frontend/src/utils/constants.tsx

```diff
@@ -107,3 +109,10 @@ export const MAX_TILES_GRID_VIEW_MOBILE = 3;
  * @constant {number} MAX_TILES_SPEAKER_VIEW_MOBILE - the maximum number of subscriber video tiles in active-speaker view on mobile
  */
 export const MAX_TILES_SPEAKER_VIEW_MOBILE = 2;
+
+/**
+ * The number of buttons used to control the Right Panel. This value determines the number
+ * of buttons shown on the right side of the Toolbar.
+ * @returns {number} The number of buttons present for the right panel
+ */
+export const RIGHT_PANEL_BUTTON_COUNT = 3 - (isReportIssueEnabled() ? 0 : 1);
```

Comment:
This is the reason why we broke out `isReportIssueEnabled`. We needed to account for the button in the right panel toolbar button count.

Reply:
Where does the `3` come from?

Reply:
That's how many buttons the user (or us I guess?) designates as right panel buttons. It enables customizability for future users, in this screenshot I change the value to `4`.
<img width="1087" alt="Screenshot 2025-03-13 at 5 33 58 PM" src="https://github.com/user-attachments/assets/8f1f237d-46ce-4b21-a152-48e41f95f217" />